backend/app/adapter/graphql/resolver/authmutation.go

```diff
@@ -59,9 +72,16 @@ func (a AuthMutation) CreateURL(args *CreateURLArgs) (*URL, error) {
 	}
 }
 
-func newAuthMutation(user *entity.User, urlCreator url.Creator) AuthMutation {
+// CreateChange creates a new Change in the ChangeLog
+func (a AuthMutation) CreateChange(args *CreateChangeArgs) (Change, error) {
+	change, err := a.changeLog.CreateChange(*args.Change.Title, args.Change.SummaryMarkdown)
```

Comment:
```suggestion
	change, err := a.changeLog.CreateChange(args.Change.Title, args.Change.SummaryMarkdown)
```
`*` means `deference`. Dereferencing pointer is dangerous because the process will crash if the pointer is `nil`.